Description: g_thread_init() is deprecated
 Remove it to fix FTBFS
Author: Ying-Chun Liu (PaulLiu) <paulliu@debian.org>
Index: clutter-gesture-0.0.2.1/clutter-gesture/clutter-gesture.c
===================================================================
--- clutter-gesture-0.0.2.1.orig/clutter-gesture/clutter-gesture.c	2012-04-02 23:24:46.302416656 +0800
+++ clutter-gesture-0.0.2.1/clutter-gesture/clutter-gesture.c	2012-04-02 23:24:52.826293634 +0800
@@ -297,8 +297,6 @@
   object_class->set_property = clutter_gesture_set_property;
   object_class->get_property = clutter_gesture_get_property;
 
-  if (!g_thread_supported ())
-    g_thread_init(NULL);
   lock = g_mutex_new();
 
   g_type_class_add_private (klass, sizeof (ClutterGesturePrivate));
